app/models/notification.rb

```diff
@@ -0,0 +1,28 @@
+class Notification < ApplicationRecord
+
+  USER_ADDED = "USER_ADDED".freeze
+  USER_DELETED = "USER_DELETED".freeze
+  ORGANIZATION_CREATED = "ORGANIZATION_CREATED".freeze
+
+  NOTIFICATION_TYPES = [USER_ADDED, USER_DELETED, ORGANIZATION_CREATED]
+
+
+  before_create :set_body
+
+  belongs_to :notificable, polymorphic: true
+  belongs_to :user
+
+  scope :latest_notifications, -> { order(id: :desc).limit(10) }
+  scope :unread, -> { where(read: false ) }
+
+  def set_body
+    case self.notification_type
+    when "USER_ADDED"
+      self.body = "User has been added to organization"
```

Comment:
extend these w/ mo details
also, better option would be to have this as reader method instead of pre-populated colum in db